### DISCUSSION TITLE
List Row + Boxed (Update)

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1167] [Proposals]
### Goal

Ter a possibilidade de trabalhar com mais variantes dentro dos Slots, o novo cenário de Assinaturas do novo APP terão novos cenários com a utilização de uma variante de Lists mais avançada.

### Context

O novo aplicativo Vivo tem a necessidade de utilização de elementos como Text, Button e Stacking Group dentro do Slot.

### Impact

Dá a possibilidade de trabalhar com variantes novas dentro das jornadas da Vivo. Evitando também de criarmos um novo componente só para este cenário que é usado em Cards. (link figma)

### Discussion Questions

Criar novas variantes no lugar do Slot de:
- Text
- Button Link
- Stacking Group (em desenvolvimento)

### Do you have a visual proposal?

https://www.figma.com/file/ACXesGuPsCXH36TEL70J9z/Lists-%2F-Advanced?node-id=106%3A1047&t=3hSFTZieqKraB2Vi-1

**Comment by yceballost** [comment]
Hola Bruno!

En principio lo que planteas no debería ser necesario crear un slot propio para esas composiciones tan sencillas, pero tampoco puedo asegurarte que para el CMS esto sea así. Sería necesario más información sobre casos de usos que se pretende usar esto.

Hasta donde llega mi conocimiento sobre el CMS, en el caso de composiciones sencillas, el CMS debería incluir el slot correspondiente sin mayores problemas. El inconveniente surgiría si este slot tuviera que combinarse con otros elementos en determinados casos de uso.

https://rebrand.ly/zpntm8u

En resumen, si el slot en cuestión tiene una funcionalidad más allá de simplemente mostrar un componente, puede que sea necesario crear un componente específico para ello. Sin embargo, esto solo se podrá determinar una vez que se haya definido un PRD que detalle los casos de uso en los que se utilizará esta funcionalidad.

Espero haberme explicado de manera clara
